docs/Remove-Item2.md

```diff
@@ -0,0 +1,136 @@
+---
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Remove-Item2
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```

Comment:
Remove an item from a location. Location is not limited by 256 character path length.